packages/react-reconciler/src/SchedulerWithReactIntegration.js

```diff
@@ -63,6 +64,7 @@ export const IdlePriority: ReactPriorityLevel = 95;
 export const NoPriority: ReactPriorityLevel = 90;
 
 export const shouldYield = Scheduler_shouldYield;
+export const requestPaint = Scheduler_requestPaint;
```

Comment:
We probably need to feature test this so it doesn’t break in RN or too terribly for mismatched versions.